webextension/js/popup.js

```diff
@@ -40,6 +40,33 @@ function showContainerTabs(containerId) {
   hideorshowIcon.src = CONTAINER_HIDE_SRC;
 }
 
+if (localStorage.getItem('onboarded2')) {
```

Comment:
@jonathanKingston - I'm really not happy with this logic for controlling the shown/hidden state of the onboarding vs. container panels. I'll keep poking at this code to improve it.